### DISCUSSION TITLE
Blog path organization

### DISCUSSION THREAD

**Discussion by reuseman** [majesticooss/astros#16] [Q&A]
Hey thanks for the template, it really looks nit.

I have a question about how you are organizing the paths for the blog. I was expecting something like this:
- mywebsite.com/blog
- mywebsite.com/blog/my-first-post
- mywebsite.com/blog/category
- mywebsite.com/blog/category/tutorial
- mywebsite.com/blog/tag
- mywebsite.com/blog/tag/astro

Is there a specific way why you organized it in the way it is?

**Comment by zanhk** [comment]
Hi @reuseman thanks you for your feedback, I just tought that shorter url looks cleaner, but I made a branch that change the behaviour like you proposed, you can checkout here: https://github.com/zankhq/astros/tree/feat/blog-paths

Preview: https://ed7faf0f.astros-7h0.pages.dev/blog

**Reply by zanhk** [reply]
In the branch I also fixed a bug so I just merged it on main so the changes are now on the main branch.

There's still a tag v0.0.4 to use the previous version
